macros/schema_tests/at_least_one.sql

```diff
@@ -1,4 +1,8 @@
 {% macro test_at_least_one(model) %}
+  {{ return(adapter.dispatch('test_at_least_one', packages = dbt_utils._get_utils_namespaces())(model, **kwargs)) }}
```

Comment:
Pleasantly surprised this doesn't need the same hack as `varargs` did. Tested locally just to be extra sure.

Reply:
yep the big trip ups were:
- passing non-kw args to a function without regular arguments, and
- assigning default values to args within a macro.
thanks for being here with me for the "Trough of Disillusionment" phase of the my personal Jinja hype cycle. it's grim!

Reply:
Jinja is pythonic, in good times & in bad